lieer/remote.py

```diff
@@ -174,6 +184,12 @@ def _cb (rid, resp, excep):
 
       try:
         batch.execute (http = self.http)
+
+      except Remote.UserRateException as ex:
+        print ("remote: user rate error, waiting %.1f seconds..")
+        time.sleep (user_rate_delay)
```

Comment:
I am seeing a lot of

------------------------------------------------------------
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..                                                                                                                                 
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
remote: user rate error, waiting %.1f seconds..
--------------------------------------

maybe this should include an exponential backoff?